pypeit/scripts/chk_wavecalib.py

```diff
@@ -29,8 +29,7 @@ def main(args):
 
         try:
             # Load
-            waveCalib = wavecalib.WaveCalib.from_file(args.input_file)
-                                                    #, chk_version=(not args.try_old))
+            waveCalib = wavecalib.WaveCalib.from_file(args.input_file, chk_version=False)
```

Comment:
do we want this to be `chk_version=True` again?

Reply:
I think we are trying to make most of these scripts (somewhat) backwards compatible. This is why I set `chk_version` to `False`